go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/celestiaorg/blobstream-contracts/v3 v3.1.0
 	github.com/celestiaorg/go-square v1.0.1
 	github.com/celestiaorg/go-square/merkle v0.0.0-20240117232118-fd78256df076
-	github.com/celestiaorg/knuu v0.13.2
+	github.com/celestiaorg/knuu v0.14.0-rc.0
```

Comment:
thanks for the pr @smuu 🙏🏻
when are you guys planning on releasing this version? I'd prefer that over using an rc.

Reply:
The latest beginning of next week.

Reply:
It uses the release `v0.14.0` now.